.github/workflows/maven.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Deploy P2 Updatesite
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
+    - name: Replace Updatesite
+      run: |
+        rm -r updatesite
+        git config --local user.email "tellastory73@gmail.com"
+        git config --local user.name "Ingo Mohr"
+        git checkout -B updatesite
+        git rm -r updatesite/
+        mkdir updatesite
+        cd releng-updatesite/target
+        mv  -v repository/* ../../updatesite/
+        cd ..
+        cd ..
+        git add updatesite/
+    - name: Commit new Updatesite
+      run: |
+        git commit -m "deploy new updatesite"
+    - name: GitHub Push
+      uses: ad-m/github-push-action@v0.6.0
+      with:
+        branch: updatesite
+        force: true
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I imagine I'll have to setup a github token/secret on the repo? :)

Reply:
Actually, I have no idea (yet) how that Github Push action determines the token. I have 4 tokens in my account, so if that action uses one of them, then: which one?
It was like that old Macintosh promise: „It just works“. 😂
I‘ll check today what token is referenced.

Reply:
Some documentation here: https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

Reply:
Ok, so the quick take-away is that every repo on Github automatically has this token once you enable Github Actions on the repo. It has some limitations, but it certainly does the thing for our small build job.

Reply:
Awesome glad to hear :)